src/observable/ArrayObservable.ts

```diff
@@ -101,7 +101,7 @@ export class ArrayObservable<T> extends Observable<T> {
   // value used if Array has one value and _isScalar
   value: any;
 
-  constructor(public array: T[], public scheduler?: Scheduler) {
+  constructor(private array: T[], private scheduler: Scheduler = null) {
```

Comment:
I tried to unify across all the create and ctors, I just went with the most common and rolled it out.